packages/cli/src/databases/migrations/sqlite/index.ts

```diff
@@ -67,6 +67,7 @@ import { CreateAnnotationTables1724753530828 } from '../common/1724753530828-Cre
 import { CreateProcessedDataTable1726606152711 } from '../common/1726606152711-CreateProcessedDataTable';
 import { SeparateExecutionCreationFromStart1727427440136 } from '../common/1727427440136-SeparateExecutionCreationFromStart';
 import { UpdateProcessedDataValueColumnToText1729607673464 } from '../common/1729607673464-UpdateProcessedDataValueColumnToText';
+import { AddProjectIcons1729607673469 } from '../common/1729607673469-AddProjectIcons';
```

Comment:
sqlite driver still re-creates tables on `addColumn` and `dropColumns`, which means this could lead to data-loss. 
Until we fix the driver/query-runner, we need to keep adding an explicit sqlite migration with `transaction: false`.
You can find an example [here](https://github.com/n8n-io/n8n/pull/11691/files#diff-8cfaa526df8da36efdf9a4e7ce5265396f668304ab7ecf8a45244a09b45f16ea).